conda-envs/windows-environment-dev-py38.yml

```diff
@@ -11,6 +11,8 @@ dependencies:
 - cloudpickle
 - fastprogress>=0.2.0
 - h5py
+- mkl==2020.4
```

Comment:
I don't think we need it for the dev environment, do we?

Reply:
No not for the builds -- but maybe the dev and test envs shouldn't be out of sync?  Not sure.  BTW I don't know why exactly this fixes things yet.  `MarginalKron` is the only place an issue occurs, so I suspect it may be a little used Aesara Op.  Will look into it further, I'm hoping that it can be fixed.

Reply:
No, for dev envs we don't want to force the BLAS backend being used.

Reply:
fixing the mkl version forces blas?